runtime-components/non-repudiation-postgresql/src/test/scala/com/daml/nonrepudiation/postgresql/NonRepudiationProxyConformance.scala

```diff
@@ -41,13 +45,19 @@ final class NonRepudiationProxyConformance
 
   behavior of "NonRepudiationProxy"
 
+  private val defaultTestsToRun = Tests
+    .default()
+    .filter {
+      case _: ClosedWorldIT => false
+      case _: CommandDeduplicationIT => false
+      case _ => true
+    }
+  private val optionalTestsToRun = Tests.optional().filter {
+    case _: KVCommandDeduplicationIT => true
+    case _ => false
+  }
   val ConformanceTestCases: Vector[LedgerTestCase] =
```

Comment:
Nit, since you're here (trying to follow the Scala style guide convention for "constants")

```suggestion
  private val conformanceTestCases: Vector[LedgerTestCase] =
```